hobbit-sharad.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+
+using namespace std;
```

Comment:
Testing commenting.

Can you explain why std is used. I cannot see it being used in the program.